opm/simulators/wells/BlackoilWellModel.hpp

```diff
@@ -416,10 +417,11 @@ template<class Scalar> class WellContributions;
             // the function handles one iteration of updating well controls and network pressures.
             // it is possible to decouple the update of well controls and network pressures further.
             // the returned two booleans are {continue_due_to_network, well_group_control_changed}, respectively
-            std::pair<bool, bool> updateWellControlsAndNetworkIteration(const bool mandatory_network_balance,
-                                                                        const bool relax_network_tolerance,
-                                                                        const double dt,
-                                                                        DeferredLogger& local_deferredLogger);
+            bool updateWellControlsAndNetworkIteration(const bool mandatory_network_balance,
+                                                       const int network_update_iteration,
+                                                       const double dt,
+                                                       bool& well_group_control_changed,
```

Comment:
Although it is not a big thing, I do not see we need to change the interface of this function.  The original form is more compact and without introducing new modified argument.